packages/@react-aria/test-utils/src/tree.ts

```diff
@@ -177,7 +203,10 @@ export class TreeTester {
         return;
       }
 
-      await this.keyboardNavigateToRow({row});
+      // TODO: We always Use Option/Ctrl when keyboard navigating so selection isn't changed
+      // in selectionmode="replace"/highlight selection when navigating to the row that the user wants
+      // to expand. Discuss if this is useful or not
+      await this.keyboardNavigateToRow({row, useAltKey: true});
```

Comment:
useful yes, maybe a different API though? instead of `useAltKey` which is specific to the implementation, we should do something like `selectionDuringNavigation=false` or something more about the intent here